packages/smooth_ui_library/lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -67,7 +67,7 @@ class SmoothAlertDialog extends StatelessWidget {
               Container(
                 height: height,
                 child: Text(
-                  title,
+                  title!,
```

Comment:
Same kind of problem: the `Text` cannot accept `null`.
If `title` is `null`, don't display the `Text` at all.